ui/slider.js

```diff
@@ -72,6 +72,7 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		this._mouseSliding = false;
 		this._animateOff = true;
 		this._handleIndex = null;
+		this._lastChangedValue = null;
```

Comment:
This is a bad name for a property that hold an index instead of a value. Let's rename it.